org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
+    "command": "sparkleshare",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "metadata": "metadata",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.mono-project.com/sources/mono/mono-4.2.2.30.tar.bz2",
+                    "sha256": "57858cd033be9915d7abdc5158c1faae8fa05757c3b7117cab3d703aa696c56b"
+                }
+            ]
+        },
+        {
+            "name": "openssh",
+            "config-opts": ["--without-pie"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.exonetric.net/pub/OpenBSD/OpenSSH/portable/openssh-7.5p1.tar.gz",
+                    "sha256": "9846e3c5fab9f0547400b4d2c017992f914222b3fd1f8eee6c7dc6bc5e59f9f0"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "config-opts": ["--with-openssl=no"],
+            "make-args": ["NO_PERL=1"],
+            "make-install-args": ["NO_PERL=1"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.14.1.tar.xz",
+                    "sha256": "6f724c6d0e9e13114ab35db6f67e1b2c1934b641e89366e6a0e37618231f2cc6"
+                },
+                {
+                    "type": "file",
+                    "path": "patches/git-lfs-2.2.1",
+                    "dest-filename": "git-lfs"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/git-lfs.patch"
+                }
+            ]
+        },
+        {
+            "name": "gtk-sharp3",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mono/gtk-sharp.git",
+                    "branch": "master",
+                    "commit": "9ef7d3f394b9d485674c59e92db24ac66fd9f16b"
```

Comment:
We generally don't want development snapshots of unreleased software. I know that is a problem with gtk#3 though...

Reply:
Distributions package gtk-sharp3 from master because there hasn't been a an official release and are fine. This is the whole point I'm bundling so I can avoid this current dependency mess. :)

Reply:
So what do you think is the way forward here?

Reply:
Let's leave it as is.